### ISSUE TITLE
Clear transcription text in editor after submitting

### ISSUE THREAD

**Issue by TimJentzsch** [GrafeasGroup/blossom#279]
This is likely caused by #277.

When starting a new transcription, the editor will contain the text of the last transcription, instead of displaying the placeholder.

**Comment by itsthejoker** [comment]
I really don't think it's because of that but I don't have any better ideas. All I did was shorten that logic without changing anything material.

**Comment by TimJentzsch** [comment]
Maybe this will fix it?
```
{% if transcription %}
    simplemde.value(`{{ transcription|safe }}`)
{% else %}
   simplemde.value("")
{% endif %}
```
My guess would be that the value is still set from the last time and never gets reset

**Comment by itsthejoker** [comment]
ohhhhhh yeah that might do it

**Comment by TimJentzsch** [comment]
I'll try implementing that

**Comment by TimJentzsch** [comment]
Closed by #282.
